platforms/swift/Tests/YttriumTests/AccountClientTests.swift

```diff
@@ -27,7 +27,8 @@ final class AccountClientTests: XCTestCase {
             entryPoint: Self.entryPoint,
             chainId: Self.chainId,
             config: config,
-            signerType: .privateKey
+            signerType: .privateKey,
+            safe: false
```

Comment:
Fix broken Swift tests broken in: https://github.com/WalletConnect/yttrium/pull/21